tao/tao_bridge/passes/tao_optimization_pass.cc

```diff
@@ -128,6 +128,12 @@ Status TaoOptimizationPass::Run(const GraphOptimizationPassOptions& options) {
     TF_RETURN_IF_ERROR(defuse_pass.Run(options));
   }
 
+  DumpGraph(options, "before_tao_clone_pass");
+  // Add Clone Constants Pass for fakequant
+  TaoCloneConstantsForBetterClusteringPass clone_pass(false, true);
```

Comment:
maybe it's time to remove these tvm related code?  @qiuxiafei @minminsun

Reply:
I'm working on it.